addon/addon.go

```diff
@@ -166,3 +175,94 @@ func NewAddonManager(kubeConfig *rest.Config, agentImage string, agentInstallAll
 
 	return addonMgr, err
 }
+
+func GetMchImage(kubeConfig *rest.Config) (string, error) {
+	kubeClient, err := client.New(kubeConfig, client.Options{})
+	if err != nil {
+		klog.Errorf("Unable to kube client: %v", err)
+
+		return "", err
+	}
+
+	var acm bool
+	acm, err = isACM(kubeClient)
+
+	if err != nil {
+		return "", err
+	}
+
+	// Not ACM
+	if !acm {
+		klog.V(2).Info("ACM not detected")
+
+		return "", nil
+	}
+
+	klog.V(2).Info("ACM detected, wait for MCH config map")
+
+	// Get ACM version
+	acmVersion, found := os.LookupEnv(OperatorVersionEnv)
+	if !found {
+		err = fmt.Errorf("%v env var not found", OperatorVersionEnv)
+		klog.Error(err)
+
+		return "", err
+	}
+
+	mchImageCMkey := types.NamespacedName{
+		Name:      "mch-image-manifest-" + acmVersion,
+		Namespace: "open-cluster-management",
+	}
```

Comment:
Since the mch image manfeist configmap could be installed in any namespace, it would be better to get it by matching the label
```
ocm-release-version=2.5.0
```